dynatrace/src/test/java/com/dynatrace/opentelemetry/metric/CumulativeToDeltaConverterTest.java

```diff
@@ -79,7 +79,8 @@ void testConvertTotalCounterToDeltaAndUpdateCacheDouble() {
         .isCloseTo(100.3, offset);
 
     try {
-      Thread.sleep(100);
+      // wait for the map entries to expire
+      Thread.sleep(120);
```

Comment:
For the tests, the expiring map is set to 100 ms. Somehow in java 8 / windows that timeout actually doesnt run out. If we wait for a little longer, it works.

Reply:
I think this failed on my windows once when I was updating the SDK as well but since I did so many changes I didn't realize it was due to that. Maybe later on we can think on a better way for this tests without having to sleep. Maybe have an internal method visible for tests that would "force" a expiration.